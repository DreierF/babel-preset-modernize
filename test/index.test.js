import { babel, dent } from './_util';
import preset from '..';

function babelPretty(code) {
	return babel(code, { presets: [preset], compact: false, retainLines: true });
}

describe('babel-preset-optimize', () => {
	it('should be a valid babel plugin', () => {
		expect(preset).toEqual(expect.any(Function));

		expect(preset()).toMatchObject({
			plugins: expect.any(Array)
		});
	});

	it('should pass a smoke test', () => {
		expect(babelPretty(dent`
			const x = 1;
		`)).toMatchSnapshot();
	});
});
