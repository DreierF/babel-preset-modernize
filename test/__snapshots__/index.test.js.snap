// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel-preset-optimize should pass a smoke test 1`] = `"const x = 1;"`;
